hepcrawl/pipelines.py

```diff
@@ -62,10 +62,10 @@ def get_media_requests(self, item, info):
                     item['file_urls']
                 )
             )
-            return [Request(x) for x in item.get(self.files_urls_field, [])]
+            return [Request(**x) for x in item.get(self.files_urls_field, [])]
         return list()
 
-    def generate_presigned_s3_url(self, path, expire=86400):
+    def generate_presigned_s3_url(self, path, expire=7776000):
```

Comment:
why?

Reply:
86400 seconds is only 24 hours. If workflow will be restarted next day because of some problem then url with 86400 seconds won't be valid anymore.